app/assets/stylesheets/workarea/storefront/content_search/components/_content_search_autocomplete.scss

```diff
@@ -0,0 +1,3 @@
+.search-autocomplete__content {
```

Comment:
this is currently also in search-autocomplete, but it makes a lot more sense here IMO...